docs/sources/metainfo/component.xml

```diff
@@ -301,6 +301,23 @@
                       </para>
                     </listitem>
                   </varlistentry>
+                  <varlistentry>
+                    <term>cockpit-manifest</term>
+                    <listitem>
+                      <para>
+                        The software can be launched from the menus of
+                        the Cockpit admin interface.  The value of the
+                        tag is the name of a <ulink
+                        url="http://cockpit-project.org/guide/latest/packages.html">Cockpit
+                        package</ulink>.
+                      </para>
+                      <para>
+                        The manifest of the package is expected to
+                        contain at least one entry for the Cockpit
+                        shell menus.
+                      </para>
```

Comment:
Can you maybe add an example of how this looks like? (So it's quick to find out what the "name of a cockpit package" is to insert.)

Reply:
I think this is already explained in the linked Cockpit documentation, no?  I think we can actually remove this  paragraph completely and trust people to do the right thing.

Reply:
I was quickly looking through the docs and it wasn't immediately clear to me what the "package name" is actually derived from. But it turns out the "Package Manifest" section is pretty clear about it.
Let's go without example then (if one goes crazy with examples, reading the docs will also become annoying, just like when there are too few examples).